plugins/janus_videoroom.c

```diff
@@ -2321,6 +2330,43 @@ void janus_videoroom_hangup_media(janus_plugin_session *handle) {
 	}
 }
 
+static void janus_videoroom_sdp_v_format_vp8(char *mline, int mline_size, int pt, int b, const char *video_mode) {
```

Comment:
Ah, cool, I didn't know about this. Is it available on all compilers, though, or is this a feature that would require something quite new?